op-chain-ops/genesis/genesis.go

```diff
@@ -58,6 +58,7 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 		TerminalTotalDifficultyPassed: true,
 		BedrockBlock:                  new(big.Int).SetUint64(uint64(config.L2GenesisBlockNumber)),
 		RegolithTime:                  config.RegolithTime(block.Time()),
+		Cel2Time:                      config.RegolithTime(block.Time()),
```

Comment:
```suggestion
		Cel2Time:                      config.Cel2Time(block.Time()),
```

Should we use the filed you added below here?

Reply:
It's not the field below, but this similarly named function: https://github.com/celo-org/optimism/blob/06a3ba098ee8dbe748ce40937699a8c343fe9c2b/op-chain-ops/genesis/config.go#L427

I didn't see a reason to duplicate it and didn't want to rename it to avoid confusion or merge conflicts.